vizro-core/src/vizro/models/_components/card.py

```diff
@@ -35,13 +34,14 @@ class Card(VizroBaseModel):
     @_log_call
     def build(self):
         text = dcc.Markdown(self.text, className="card_text", dangerously_allow_html=False, id=self.id)
-        button = html.Div(
-            dbc.Button(
-                href=get_relative_path(self.href) if self.href.startswith("/") else self.href, className="card_button"
-            ),
-            className="button_container",
+        card_content = (
+            dcc.Link(
+                text,
+                href=get_relative_path(self.href) if self.href.startswith("/") else self.href,
+                className="card-link",
+            )
+            if self.href
```

Comment:
It does not have to be part of this PR, but it would be good if we could provide card.href with similar validation we have for page.path. At this moment, if user has path with an empty space it will be replaced with "-". But if users tries to add the same path to the card.href it will not work and will lead to 404 page.

Reply:
Yes, we are already aware! @l0uden already raised the issue [here](https://github.com/mckinsey/vizro/issues/285) - it's a ticket that's currently deprioritized but that anyone in the team can take on when they like and have capacity :)